contrib/resources/translations/it.lng

```diff
@@ -1348,16 +1348,16 @@ Nessuna ROM è attualmente caricata
 s
 .
 :MT32_ROMS_LABEL
-ROM  MT-32  
+ROM  MT-32
```

Comment:
Not sure if the trailing whitespace in these labels is intentional, if it is I'll roll back.

Reply:
Yes; the whitespace isn't needed and can be stripped; thanks for this cleanup.

(When padding or aligning, we'll do it programmatically using `%` printf format specifiers.)